feat: add raspberry pi spport

miettal opened the pull request:
##### SUMMARY
in case raspberry pi, role scripts mis-detect os type as debian and try to add debian repo url.
This PR change the detection to detect rasberry pi os to get right repo url.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
{agent, javagateway, web, server} role

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below

```

pyrodie18 reviewed (changes requested):
Please add a change fragment and fix the tests that are failing.  I'll be honest, I am not familiar with the `ansible_lsb.id` variable and from the little bit of reading I think it may well cause more problems than its worth just to allow Pi integration which is not something we will officially support since there's no way for us to test the integration on it.  @D3DeFi  what are your thoughts?

miettal commented:
I fixed code and added a fragment.
but Now it seems ci was failed by another reason.

> ERROR! Unknown error when attempting to call Galaxy at 'https://galaxy.ansible.com/api/v3/collections/community/general/versions/5.4.0/': The read operation timed out

miettal commented:
@pyrodie18
I'm almost done, but I cannnot pass CI test.
It seems failed test is not related my change.
What I need to do? Can I help something?

BGmot commented:
It looks like it cannot fine the package https://github.com/ansible-collections/community.zabbix/actions/runs/7307787096/job/19913806734?pr=1145#step:7:228
Can you please take a close look?

miettal commented:
in centos script, mysql(maria db) 10.8 repo link is hardcorded in https://github.com/ansible-collections/community.zabbix/blob/31d49d6ba07e3a0dd5b0ea1e70eea0527c61fa9e/roles/zabbix_proxy/tasks/RedHat.yml#L103.
And the url https://mirror.rackspace.com/mariadb/yum/10.8/centos7-amd64/repodata/repomd.xml return 404.

I cheked server in my browser, It seems 10.8 was dropped and 10.8 reached to EOL.
![スクリーンショット 2023-12-25 12 24 18](https://github.com/ansible-collections/community.zabbix/assets/118803/8239cc58-0dcb-4b8a-98f0-dc157da78460)
![スクリーンショット 2023-12-25 12 24 45](https://github.com/ansible-collections/community.zabbix/assets/118803/9f3f4eda-6068-4598-a366-759a0c096d30)

So I guess this CI fails is not related to my change and In order to pass CI changing mysql version in censto script is needed.
